godotproject/Menus/InGameMenu/PauseSubmenu.gd

```diff
@@ -0,0 +1,47 @@
+extends MarginContainer
+
+export (NodePath) var first_focus_path
+onready var first_focus = get_node(first_focus_path)
+export (NodePath) var menu_parent_path
+onready var menu_parent = get_node(menu_parent_path)
+
+const fade_transition = false
+
+const debug_mode_inputs = [
+	2, 1, 3, 0,
+	2, 3, 0, 1,
+	1, 0, 2, 3,
+	0, 1, 2, 3,
+	2, 0, 1, 3
+]
+var debug_mode_input_index = 0
+
+func set_initial_focus():
+	first_focus.grab_focus()
+
+func activate_debug_mode():
+	print("Debug mode would be activated")
+
+func _process(_delta):
+	if Input.is_action_pressed("ui_cancel"):
+		var debug_mode_current_input = -1
```

Comment:
can we delete all this debug stuff or does it still need to be here?

Reply:
My plan for the "debug mode" was to basically have a set of inputs that we can put in on the pause menu that activates a debug mode where we can do administration tasks like properly shutting down the Pi. I did the code that would open up the debug menu but haven't actually made it do anything.

Reply:
ok cool could also be useful for enabling the players god_mode for testing purposes.